catalogue/webapp/pages/item.js

```diff
@@ -48,9 +49,7 @@ const ItemPage = ({
       hideNewsletterPromo={true}
     >
       <Layout12>
-        <h1>{title}</h1>
```

Comment:
What's happened to the title?

Reply:
Oh yeah – I wanted to tidy up the UI, but forgot to put this back. Might put it below the visuals.